### PR TITLE
Envy search goes to Google

### PR DESCRIPTION
**Pull request by WindowsMI**
So I was just exploring Envy's source code when I found out this:

"You can put Google in an iframe if you add the parameters "webhp?igu=1" to the end of the link."

So I did that and the Envy search worked just fine :)!

(STILL DRAFT THO)

### CODE REVIEW

**Comment by WindowsMI** [comment]
out of draft :)

**Comment by DavidTDC3377** [comment]
What benefit does this give? We could still use Bing so I don't really see a point of switching search engines. They do the same thing, _search for stuff_

File: scripts.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 if (/Mobi/.test(navigator.userAgent) === true) location.replace('mobile/index.html')
 
   function SearchQuery(val) {
-    document.getElementById("searchButton").href = "https://www.bing.com/search?q=" + val;
+    document.getElementById("searchButton").href = "https://www.google.com/search?q=" + val + "&sca_esv=564974394&igu=1&source=hp&ei=kYgBZZ25Ebne2roP5Laq0Ac&iflsig=AD69kcEAAAAAZQGWodG9Ayd0DXLzPMg1NHZHGSXpjRXz&ved=0ahUKEwjdxeuwqqeBAxU5r1YBHWSbCnoQ4dUDCAk&uact=5&oq=hi&gs_lp=Egdnd3Mtd2l6IgJoaTIHECMYigUYJzIIEAAYigUYkQIyCBAAGIoFGJECMgUQABiABDILEC4YgAQYxwEYrwEyBRAuGIAEMgUQABiABDIFEAAYgAQyBRAAGIAEMgUQLhiABEjIClAAWLYBcAB4AJABAJgB2gGgAfECqgEFMC4xLjG4AQPIAQD4AQHCAgQQIxgnwgIIEC4YigUYkQI&sclient=gws-wiz"
```
Comment (DavidTDC3377) [inline scripts.js:6]: What is this long line for?

**Review by DavidTDC3377 — COMMENTED** [review]
There are merge conflicts anyway, so I'll maybe manually add it

**Comment by OwnedByWuigi** [comment]
Adding right now, Bing has always been a wall of ads for me, and people are put off that we use Bing/FrogFind

**Comment by DavidTDC3377** [comment]
Done, [{commit}](https://github.com/envywebos/envywebos.github.io/commit/93fc50a13c91b34cb71f4107e8a8cf8d58ff35ad)

**Pull request closed without merging**